src/pages/GameSelectionScreen.vue

```diff
@@ -326,7 +326,8 @@ export default class GameSelectionScreen extends Vue {
         await this.$store.dispatch('checkMigrations');
         this.runningMigration = false;
 
-        await this.$store.dispatch('resetProfileModule');
+        await this.$store.dispatch('resetLocalState');
```

Comment:
In my earlier review I recommended making either all the resets either async/dispatch/action or sync/commit/mutation. That is not a hard requirement, just a preference. However, now the PR mixes the two approaches, and IDK if that's intentional or an oversight. So from communication perspective it would be nice if you'd left a comment stating why you decided to do it this way.

Reply:
In my understanding commits are for "directly" modifying the state, dispatches are for launching other commits (or dispatches). So, in this case we're launching a few data resets from index.ts, so a commit won't do it, but in other places we're just resetting one module at a time and thus using the commit directly. I might be wrong though,

Reply:
You're not wrong, although I would say that the main point of actions is to be able to execute "external" async code before/after doing the commits. (And yes, due to technical limitations mutation can't mutate the state of sub/sibling stores).

But I guess my point is somewhere along these lines: now we have one line of code on GameSelectionScreen calling this action, and then six lines defining the action in the index store. We could instead have just four lines of code calling the commits directly on the GameSelectionScreen, with the added benefit of not having to jump to the wrapper action to see what it does. And yes, this somewhat contradicts my earlier comment about doing it exactly like it's done here, but that was done under the assumption that it would avoid some repetition when we could call the wrapper from multiple places (but now we don't since we do a bit different thing on Profiles.vue).

All that being said, **I don't consider this a blocker** as long as the choice to do it this was is deliberate.